SConstruct

```diff
@@ -1332,10 +1332,9 @@ sanitizers = env.ParseList(GetOption('sanitizers'), supported_sanitizers)
 if sanitizers:
     if not compiler in ['gcc', 'clang']:
         env.Die("sanitizers are not supported for compiler '%s'" % compiler)
-
+    
     for name in sanitizers:
-        flags = ['-fsanitize=%s' % name]
-
+        flags = ['-fsanitize=%s' % name, '-fno-sanitize-recover=%s' % name]
```

Comment:
https://github.com/roc-streaming/roc-toolkit/pull/412#discussion_r455096510

Don't we want to specify `-fno-sanitize-recover` only once with a list?

Reply:
Checked it, passing multiple options is also ok.